pkg/sensors/tracing/generickprobe.go

```diff
@@ -540,17 +540,17 @@ func createGenericKprobeSensor(
 		selMaps:       selMaps,
 	}
 
-	for i := range kprobes {
-		syms, syscall, err := getKprobeSymbols(kprobes[i].Call, kprobes[i].Syscall, lists)
+	for _, kprobe := range kprobes {
```

Comment:
Is this correct?

`kprobes` is `[]KprobeSpec` so not a pointer, so kprobe will be a copy and when we update it, the original array element will not change. Not sure if this is an issue since none of the tests are failing, but it is different from the original behavior where we use `kprobe[i]` (e.g., `kprobe[i].Syscall = syscall` actually modifies the array element).

Maybe do:
```
for i := range kprobes {
       kprobe = kprobes[i]
}
```

To be on the safe side?

Reply:
yes, but that kprobe is used only in that loop, so I think it's ok.. unless we use that kprobe def elsewhere.. in which case it's safer to keep it as it was.. ;-) I'll remove that commit to be on safe side as you said

Reply:
Ah, OK.

BTW, I meant doing `kprobe = &kprobes[i]`, but does not matter :)